recipes/cuda-cccl-impl/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "cuda-cccl-impl" %}
+{% set version = "2.0.1" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.gz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/NVIDIA/thrust/archive/refs/tags/{{ version }}.{{ extension }}
+    folder: thrust
+    sha256: 06df373cf487cf3710be6798bc5441a03685b78e5c1338e2f14005f7ccd030e6  # [linux]
+    sha256: 39a9fc88b0b06da118cc3ed28cbfca27b2a7ae192877996cab1295176ceed65e  # [win]
+  - url: https://github.com/NVIDIA/cub/archive/refs/tags/{{ version }}.{{ extension }}
+    folder: cub
+    sha256: beeef0962c87eca994db06cb9e9168a71c63fb369398a5b5f1db923140814701  # [linux]
+    sha256: 22308facf62421d6dd2489b9554ffa4908dca13d3921a3e7f2965c5a6984cd20  # [win]
+  - url: https://github.com/NVIDIA/libcudacxx/archive/refs/tags/1.9.0.{{ extension }}
```

Comment:
I think we should use a version tag for this instead of hardcoding `1.9.0`?

Reply:
Added tags for the 3 components now. This will get cleaned up once the repos are merged and we only have 1 version here.